src/EventExport/Format/HTML/HTMLEventFormatter.php

```diff
@@ -199,7 +199,9 @@ private function addCalendarInfo($eventId, stdClass $event, array &$formattedEve
     {
         if ($this->calendarSummaryRepository) {
             try {
-                $calendarSummary = $this->calendarSummaryRepository->get($eventId, ContentType::HTML(), Format::SMALL());
+                $calendarType = $event->calendarType;
+                $calendarSummaryFormat = ($calendarType == "multiple" ? Format::SMALL() : Format::LARGE());
```

Comment:
```suggestion
                $calendarSummaryFormat = $calendarType === "multiple" ? Format::SMALL() : Format::LARGE();
```

Reply:
Maybe also use the `CalendarType` enum instead of hardcoding `multiple` as a string here.
So something like `CalendarType::fromNative($calendarType)->is(CalendarType::MULTIPLE())`.